syncano_cli/parse_to_syncano/migrations/transfer.py

```diff
@@ -97,6 +98,8 @@ def transfer_objects(self, instance):
                     objects_to_add.append(batched_syncano_object)
                     parse_ids.append(data_object['objectId'])
 
+                    time.sleep(6)  # avoid throttling;
```

Comment:
But we only have limits per second. So every second it's a fresh limit. So time.sleep(1) should always "refresh" it. And this way it will be only possible to run into rate limit when we use more than 60 calls before sleeping